docs/_sass/overview.scss

```diff
@@ -34,15 +34,20 @@ h2[id]:before, h4[id]:before, h5[id]:before {
     height: 6rem;
     visibility: hidden;
 }
+
+#search-results h4[id]:before {
```

Comment:
is this change for the safari specific dot? wherever you are doing that change please mention in comments ,
that this is for safari extraneous dot appearance.

Reply:
@legendarykamal just to confirm. You want me to mention that I am making the change for safari in this pull request comment? or do you mean in the code comment?

Reply:
Commenting in the code is helpful in terms of sustainability.